### PR TITLE
ms1106: add 3 new fields

### PR DESCRIPTION
**Pull request by MingtaoSun**
http://ourweb.nla.gov.au/apps/jira/browse/DSCS-1106
@scoen @shuangzhou @bsingh-nla 
Add 3 new fields

### CODE REVIEW

File: banjo/conf/schema.xml [hunk 1]
```diff
@@ -205,7 +205,10 @@
    <field name="acknowledgement_party" type="string" indexed="true" stored="true" multiValued="true"/>
    <field name="acknowledgement_type" type="string" indexed="true" stored="true" multiValued="true"/>
    <field name="work_created_during_migration" type="boolean" indexed="true" stored="true"/>
-   
+   <field name="additional_series_statement" type="text_general" indexed="true" stored="true"/>
+   <field name="sheet_name" type="text_general" indexed="true" stored="true"/>
+   <field name="sheet_creation_date" type="int" indexed="true" stored="true"/>
```
Comment (bsingh-nla) [inline banjo/conf/schema.xml]: why not type="date" ?

Comment (MingtaoSun) [inline banjo/conf/schema.xml]: just followed existing example, expiry_date, start_date, end_date, all in the type of int. I don't think there is such type as date in schema.

Comment (bsingh-nla) [inline banjo/conf/schema.xml]: I think most of the date/time fields got type="date" e.g. issue_date, start_date, end_date etc. I think int might be only used if date/time is in milliseconds.

Comment (bsingh-nla) [inline banjo/conf/schema.xml]: expiry_date only got year :smile: 

http://hammer:10330/solr/#/banjo/query
data from solr 
"expiry_date": 2034,
"digital_status_date": "2014-05-22T04:30:10.718Z",

Comment (MingtaoSun) [inline banjo/conf/schema.xml]: @bsingh-nla , you are right Singh. I must have been blind :sob:

File: banjo/conf/schema.xml [hunk 2]
```diff
@@ -205,7 +205,10 @@
    <field name="acknowledgement_party" type="string" indexed="true" stored="true" multiValued="true"/>
    <field name="acknowledgement_type" type="string" indexed="true" stored="true" multiValued="true"/>
    <field name="work_created_during_migration" type="boolean" indexed="true" stored="true"/>
-   
+   <field name="additional_series_statement" type="text_general" indexed="true" stored="true"/>
+   <field name="sheet_name" type="text_general" indexed="true" stored="true"/>
+   <field name="sheet_creation_date" type="date" indexed="true" stored="true"/>
```
Comment (MingtaoSun) [inline banjo/conf/schema.xml]: @bsingh-nla , fixed.

**Pull request merged**